.github/workflows/build_wamr_lldb.yml

```diff
@@ -63,7 +63,8 @@ jobs:
       - name: install utils macos
         if: steps.lldb_build_cache.outputs.cache-hit != 'true' && contains(inputs.runner, 'macos')
         run: |
-          brew install swig cmake ninja libedit
+          brew remove swig
```

Comment:
Is there a better way to remove a pre-installed dependency from a GitHub image?

Reply:
As far as I know, there isn't a better way(. I think removing the existing version and reinstalling the version needed is a solid way.